src/pages/komodo-defi-framework/api/common_structures/wallet/index.mdx

```diff
@@ -62,3 +67,47 @@ The `InputTxns` object includes the following items:
     }
   ```
 </CollapsibleSection>
+
+### PayForGas
+
+The `PayForGas` object includes the following items:
+
+| Parameter                    | Type    | Description                                                                                                                                                                  |
+| ---------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| tx\_type                     | string  | ERC20-like coins and tokens only. Options are `Legacy` or `Eip1559`. The type of transaction values being configured.                                                        |
+| gas\_price                   | decimal | Values are in Gwei. The base gas price for the transaction.                                                                                                                  |
```

Comment:
gas_price is for tx_type = Legacy, this is max price per gas unit the user is willing to pay for the transaction